engine/src/main/java/org/terasology/rendering/dag/nodes/FirstPersonViewNode.java

```diff
@@ -114,4 +117,9 @@ public void process() {
 
             PerformanceMonitor.endActivity();
     }
+
+    @Override
+    public void update() {
+        readOnlyGBufferFbo = displayResolutionDependentFBOs.get(READONLY_GBUFFER);
```

Comment:
I might be wrong but this looks like another case where we are not actually using this and readOnlyGBufferFbo is not necessary?